src/containers/Home/Home.js

```diff
@@ -63,78 +93,54 @@ export default class Home extends Component {
+                  <li><span className="fa fa-search" /> Pagination, filtering and sorting</li>
                   <li><span className="fa fa-magic" /> Generate the data model using <a href="https://schema.org" target="_blank">Schema.org</a></li>
-                  <li><span className="fa fa-user" /> <a href="http://symfony.com/doc/current/bundles/FOSUserBundle/index.html">FOSUser</a> integration</li>
-                  <li><span className="fa fa-key" /> <a href="https://jwt.io/" target="_blank">JWT</a> and <a href="https://oauth.net/" target="_blank">OAuth</a> support</li>
+                  <li><span className="fa fa-user" /> <a href="http://symfony.com/doc/current/bundles/FOSUserBundle/index.html">FOSUser</a>,  <a href="https://jwt.io/" target="_blank">JWT</a>, CORS and <a href="https://oauth.net/" target="_blank">OAuth</a> support</li>
                   <li><span className="fa fa-lock" /> Implements <a href="https://www.owasp.org/index.php/REST_Security_Cheat_Sheet" target="_blank">OWASP's recos</a></li>
-                  <li><span className="fa fa-wrench" /> Easily customizable (events, DI) and configurable</li>
                   <li><span className="fa fa-cube" /> Modular</li>
                   <li><span className="fa fa-space-shuttle" /> Designed for speed and caching</li>
                   <li><span className="fa fa-check" /> <a href="http://behat.org/" target="_blank">Behat</a>, <a href="http://phpunit.de" target="_blank">PHPUnit</a> and <a href="https://www.getpostman.com/" target="_blank">Postman</a> spec &amp; testing</li>
-                  <li><span className="fa fa-html5" /> Works with all JS frameworks (<a href="https://facebook.github.io/react/" target="_blank">React</a>, <a href="https://angular.io/" target="_blank">Angular</a>...)</li>
                   <li><span className="fa fa-legal" /> 100% open source (MIT)</li>
                 </ul>
               </div>
             </div>
           </section>
         </div>
 
-        <div id="architecture" className="slide">
+        <div id="architecture" className="architecture slide">
           <section className="container">
-            <h1>The web has changed!</h1>
+            <h2>The web has changed!</h2>
 
             <ul>
               <li>
-                <b>Javascript webapps (<a href="https://en.wikipedia.org/wiki/Single-page_application" target="_blank">SPA</a></b>) are the standard:
-                <div>
-                  Awesome frontend technologies such as <a href="http://facebook.github.io/react/" target="_blank">React</a> and <a href="http://angular.io" target="_blank">Angular</a> help to create them.<br />
-                  Thanks to <a href="http://redux.js.org/docs/recipes/ServerRendering.html" target="_blank">Universal Rendering</a>, they are now indexed by search engines.
-                </div>
+                <h3>Javascript webapps (<a href="https://en.wikipedia.org/wiki/Single-page_application" target="_blank">SPA</a>) are the standard.</h3>
```

Comment:
It's about structuring content importance. IMHO it's more important than a basic paragraph but I may be wrong